ui/app/src/views/config/ConfigView.tsx

```diff
@@ -0,0 +1,38 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Box, Stack, Typography } from '@mui/material';
+import Cog from 'mdi-material-ui/Cog';
+import { JSONEditor } from '@perses-dev/components';
+import AppBreadcrumbs from '../../components/AppBreadcrumbs';
+import { useConfig } from '../../model/config-client';
+
+function ConfigView() {
+  const cfg = useConfig();
```

Comment:
I think we should add loading support:
```suggestion
  const { data, isLoading } = useConfig();
```

Reply:
And add loading element: 
```
if (isLoading) {
    return (
      <Stack width="100%" sx={{ alignItems: 'center', justifyContent: 'center' }}>
        <CircularProgress />
      </Stack>
    );
  }
```